src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,61 @@
+package Duke;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+/**
+ * A Bot that keeps track of the user's task list.
+ * Task List can take in three types of tasks:
+ * 1. Todo
+ * 2. Deadline
+ * 3. Event
+ * Task List is saved into a File called duke.txt in the computer.
+ */
+
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private Recieve recieve;
+    private Parser parser;
+
+    /**
+     * A public constructor to initialize a session with Duke.
+     *
+     * @throws IOException Thrown when the file is not found.
+     */
+    public Duke() throws IOException {
+        String dukeFile = "duke.txt";
+        storage = new Storage(dukeFile);
+        try {
+            tasks = new TaskList(storage.load());
+            parser = new Parser(tasks, storage);
+            recieve = new Recieve(parser);
```

Comment:
Minor spelling error: `recieve` → `receive`
In IntelliJ, you could refactor>rename to mass rename the variable name.